### PR TITLE
vgremove: add page

### PR DESCRIPTION
**Pull request by jackiesogi**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known): LVM version: 2.03.11(2) (2021-01-08); Library version: 1.02.175 (2021-01-08)**

### CODE REVIEW

File: pages/linux/vgremove.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+
+`vgremove my_volume_group`
+
+- Forcefully remove a volume group without confirmation:
+
+`vgremove --force my_volume_group`
+
+- Set the debug level for detailed logging (level 2):
+
+`vgremove --debug 2 my_volume_group`
```
[the remark this answers is not in the document]
Comment (jackiesogi) [reply pages/linux/vgremove.md]: I misunderstood the documentation!!!
-d means level 1 debug info and -dd means level 2... and so on.

@sebastiaanspeck What do you think is better way to demonstrate this example?

File: pages/linux/vgremove.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
+# vgremove
+
+> Remove volume group(s) in LVM.
+> More information: <https://manned.org/vgremove.8>.
```
Comment (spageektti) [inline pages/linux/vgremove.md:4]:
```suggestion
> More information: <https://manned.org/vgremove>.
```
The page is the same, but the URL is shorter.

**Review by spageektti — COMMENTED** [review]
Thanks for your contribution

File: pages/linux/vgremove.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Waples) [reply pages/linux/vgremove.md]: Just to stop you there, I don't want to see `--debug --debug --debug --debug --debug --debug --debug`

As implied in the 2nd part of your remark.

**Review by spageektti — APPROVED** [review]

**Pull request merged**